src/extractor/CaretValueRuleExtractor.ts

```diff
@@ -231,6 +240,29 @@ export class CaretValueRuleExtractor {
     return caretValueRules;
   }
 
+  static processConcept(
+    input: any,
+    conceptHeirarchy: string[],
+    codeSystem: ProcessableCodeSystem,
+    fisher: utils.Fishable
+  ): ExportableCaretValueRule[] {
+    const caretValueRules = this.processResource(input, fisher, 'Concept');
+    caretValueRules.forEach((rule, index) => {
+      rule.pathArray = conceptHeirarchy;
+      if (isFSHValueEmpty(rule.value)) {
+        logger.error(
+          `Value in CodeSytem ${
+            codeSystem.name ?? codeSystem.id
+          } at concept ${conceptHeirarchy.join('.')} for element ${
+            rule.caretPath
+          } is empty. No caret value rule will be created.`
+        );
+        delete caretValueRules[index];
+      }
```

Comment:
Could you add a test case for an empty value in a concept's caret rule? I don't think any of the current test cases will go into this block.

Reply:
Added in commit `a1b231b`, along with some other unit tests.